lectures/markov_chains.md

```diff
@@ -1348,13 +1343,13 @@ for x0 in range(8):
     X_bar = (X == x0).cumsum() / (1 + np.arange(N, dtype=float))
     ax.plot(X_bar - ψ_star[x0], label=f'$X = {x0+1} $')
     ax.set_xlabel('t')
-    ax.set_ylabel(fr'average time spent in a state $- \psi^* (X=x)$')
+    ax.set_ylabel(fr'average time spent in a state $- \psi^* (x)$')
 
 ax.legend()
 plt.show()
 ```
 
-We can see that the time spent at each state quickly converges to the stationary distribution.
+Note that the average time spent at each state quickly converges to the stationary distribution.
```

Comment:
Note that the fraction of time spent at each state quickly converges to the probability assigned to that state by the stationary distribution.